src/validators/formatValidators.js

```diff
@@ -0,0 +1,78 @@
+const Decimal = require('decimal.js');
+
+const constraints = {
+    int64: {
+        max: new Decimal('9223372036854775807'),
+        min: new Decimal('-9223372036854775808')
+    },
+    float: {
+        max: new Decimal(2).pow(128),
+        min: new Decimal(2).pow(128).negated()
+    },
+    double: {
+        max: new Decimal(2).pow(1024),
+        min: new Decimal(2).pow(1024).negated()
+    },
+    byte: {
+        regex: /^[a-zA-Z0-9/+]+={0,2}$/
+    }
+};
+
+module.exports.defaultFormatsValidators = [
```

Comment:
It's considered a good practice to separate export definition from actual implementation, see Revealing Module Pattern: https://weblogs.asp.net/dwahlin/techniques-strategies-and-patterns-for-structuring-javascript-code-revealing-module-pattern

Reply:
Thanks @kibertoad, read your link about revealing pattern and I'm not sure that it fits the  need here, as all I do here is exporting a const.

Isn't changing the export to `module.exports = { defaultFormatsValidators  }` would be enough?

Reply:
> Isn't changing the export to module.exports = { defaultFormatsValidators } would be enough?

Yup, that's exactly what I'm suggesting :D. Sorry for the confusion.